x/distribution/client/proposal_handler.go

```diff
@@ -0,0 +1,10 @@
+package client
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/distribution/client/cli"
+	"github.com/cosmos/cosmos-sdk/x/distribution/client/rest"
+	govclient "github.com/cosmos/cosmos-sdk/x/gov/client"
+)
+
+// param change proposal handler
+var ProposalHandler = govclient.NewProposalHandler(cli.GetCmdSubmitProposal(), rest.ProposalRESTHandler)
```

Comment:
Should this not be a constant?

Reply:
golang won't allow for this to be a constant (only specific types such as `string` can use `const`)